scapy/arch/pcapdnet.py

```diff
@@ -37,10 +37,11 @@ def winpcapy_get_if_list():
         return ret
       finally:
         pcap_freealldevs(devs)
-
   except OSError as e:
     if conf.interactive:
-      log_loading.error("Unable to import libpcap library: %s" % e)
+      log_loading.warning("wpcap.dll is not installed. You won't be able to send/recieve packets. Visit https://www.winpcap.org/ to install it")
```

Comment:
I think that it is better to refer to Scapy documentation, especially because wpcap.dll could be installed differently.